### DISCUSSION TITLE
Career Opportunities

### DISCUSSION THREAD

**Discussion by JayR1031** [dailydotdev/daily#970] [Feature Requests]
Can you include a section with career opportunities and make it free for employers to use so that software engineers and developers can quickly access these job postings and apply or reshare on your discord channel?

**Comment by idoshamun** [comment]
Hi @Jairo1031,

Thanks for the suggestion! It's currently out of our scope but we may consider in the future

**Comment by JayR1031** [comment]
My pleasure! Appreciate the response. If you have any junior developer
roles please consider me. I'm willing and able :)

Best regards,

Jairo

**Reply by idoshamun** [reply]
Unfortunately, we don't have any junior positions at the moment.
Stay tuned for new positions here: https://daily.dev/careers

**Comment by JayR1031** [comment]
Awesome, thanks for the update!
